### DISCUSSION TITLE
Question: Is there a way to persist captures and options provided to the requests?

### DISCUSSION THREAD

**Discussion by bartoszb-rgb** [jellydn/hurl.nvim#206] [Q&A]

**Comment by sweep-ai-deprecated** [comment]
❌ The ticket to PR workflow is no longer supported. We have moved to a new chat interface in private beta. Please contact us at team@sweep.dev to try it out.

:book: For more information on how to use Sweep, please read our [documentation](https://docs.sweep.dev).



<div align="right"><sup><sup><em>Tracking ID: b6e53873ea</em></sup></sup></div>

**Comment by jellydn** [comment]
Hi @bartoszb-rgb Could you explain your workflow/usage?

**Comment by bartoszb-rgb** [comment]
Hey, I usually define multiple endpoints in single file to retrieve necessary data. I specify a lots of captures. Every time I need to check last output I need to call them all to collect necessary data.
It would be perfect if I could only fire the call I need with variables from previous calls.

**Reply by jellydn** [reply]
Could you try v2 as the usage on my PR? https://github.com/jellydn/hurl.nvim/pull/207

**Comment by jellydn** [comment]
re. > It would be perfect if I could only fire the call I need with variables from previous calls.

You could define the variable with Hurl commands based on your test in your case. I have similar workflow so this the way for me now.

```lua
      { "<leader>hg", ":HurlSetVariable", desc = "Add global variable" },
      { "<leader>hG", "<cmd>HurlManageVariable<CR>", desc = "Manage global variable" },
```

If there is better idea, please open PR :) Thanks.

**Comment by jellydn** [comment]
Just FYI - https://github.com/jellydn/hurl.nvim/issues/149 PR's welcome :)
